howto/stream/oob-data.md

```diff
@@ -106,15 +106,132 @@ A trivial example to simulate the data transmission between an Anbox container a
    anbox cloud <--  the received data
    ```
 
-### Anbox data proxy
+### Anbox WebRTC data proxy
 
 To build up the communication bridge between an Android application and the Anbox WebRTC platform, you must implement a service or daemon that is responsible for:
 
  - The socket connection
  - Polling events
  - Forwarding data between both ends
 
-A system service named `anbox-data-proxy`, which covers all the above functionalities, will be implemented in the 1.16.0 release of Anbox Cloud. It makes use of [`gbinder`](https://github.com/mer-hybris/libgbinder) to set up a communication bridge and provide reliable and high-efficiency data transmission between the Android system and the Anbox WebRTC platform.
+Meanwhile, a system daemon named `anbox-webrtc-data-proxy` running in the Anbox container registers an Android system service `org.anbox.webrtc.IDataProxyService` which allows Android developers to easily make use of binder IPC for data communication between an Android application and Anbox WebRTC platform through a file descriptor.
+
+[note type="information" status="Note"]To interact with `org.anbox.webrtc.IDataProxyService` system service, the Android application must be [installed as a system app](https://discourse.ubuntu.com/t/how-to-install-an-apk-as-a-system-app/27086). [/note]
+
+
+#### Access Data Proxy Service
+
+There are two ways to access the `org.anbox.webrtc.IDataProxyService` from an Android application.
+
+- If you develop the application with Android studio, you can access the service by using the Android's reflection API.
+
+```
+    IBinder getDataProxyService() {
+        IBinder service = null;
+        try {
+            Method method = Class.forName("android.os.ServiceManager").getMethod("getService", String.class);
+            service = (IBinder) method.invoke(null, "org.anbox.webrtc.IDataProxyService");
+        } catch (NoSuchMethodException e) {
+            e.printStackTrace();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
+        } catch (InvocationTargetException e) {
+            e.printStackTrace();
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        return service;
+    }
+```
```

Comment:
This code block must be indented to be part of the bullet list.
(If this causes issues with the spell check, you might need to remove the empty line in the code.)